doc/user_guide/big_data.rst

```diff
@@ -162,6 +162,33 @@ performed lazily. Important points are:
 Chunking
 ^^^^^^^^
 
+Data saved in the HDF5 format is typically divided into smaller chunks which can be loaded separately into memory, 
+allowing lazy loading. Chunk size can dramatically affect the speed of various HyperSpy algorithms, so chunk size is
+worth careful consideration when saving a signal. HyperSpy's default chunking sizes are probably not optimal
+for a given data analysis technique. For more comprehensible documentation on chunking,
+see the dask `array chunks
+<https://docs.dask.org/en/latest/array-chunks.html>`_ and `best practices
+<https://docs.dask.org/en/latest/array-best-practices.html>`_ docs. The chunks saved into HDF5 will
+match the dask array chunks in ``s.data.chunks`` when lazy loading.
+Chunk shape should follow the axes order of the numpy shape (``s.data.shape``), not the hyperspy shape.
+The following example shows how to chunk one of the two navigation dimensions into smaller chunks:
+
+.. code-block:: python
+
+    >>> import dask.array as da
+    >>> data = da.random.random((10,200,300))
+    >>> data.chunksize
+    (10, 200, 300)
+    
+    >>> s = hs.signals.Signal1D(data)
+    >>> s # Note the reversed order of navigation dimensions
+    <Signal1D, title: , dimensions: (200, 10|300)>
+    
+    >>> s.save('chunked_signal.hspy', chunks=(10, 100, 300)) # Chunking first hyperspy dimension (second array dimension)
+    >>> s2 = hs.load('chunked_signal.hspy', lazy=True)
+    >>> s2.data.chunksize
+    (10, 100, 30)
```

Comment:
```suggestion
    (10, 100, 300)
```

Reply:
Good catch!
I've added a check to see if szip support is present (h5py docs say its dependent on the hdf5 install due to legal issues). Check the latest commit to see the changes.

Reply:
I just saw that I could accept your code change directly here. That's a nice feature, but I'd already commited it :)